src/engraving/rendering/stable/measurelayout.cpp

```diff
@@ -991,7 +991,7 @@ void MeasureLayout::cmdUpdateNotes(const Measure* measure, const DomAccessor& do
                 for (track_idx_t t = track; t < endTrack; ++t) {
                     Chord* chord = item_cast<Chord*>(segment.element(t), CastMode::MAYBE_BAD); // maybe Rest
                     if (chord) {
-                        chord->cmdUpdateNotes(&as);
+                        chord->cmdUpdateNotes(&as, chord->staffIdx());
```

Comment:
This should be `staffIdx`, not `chord->staffIdx()` (it doesn't really matter since doesn't get executed but still better to keep it correct)